org.eclipse.ice.commands/src/main/java/org/eclipse/ice/commands/RemoteMoveFileCommand.java

```diff
@@ -27,12 +26,12 @@
 	/**
 	 * The path to the source file which is to be copied
 	 */
-	Path source;
+	String source;
 
 	/**
 	 * The path of the destination for which the source file will be copied to
 	 */
-	Path destination;
+	String destination;
```

Comment:
Should be private/protected

Reply:
Whoops, sorry. I meant to go through and double check this yesterday and it slipped my mind after I talked with Jay. I've gone through and double checked all member variables are private/protected in all of the classes.

Reply:
Conversations with me tend to have that effect...